demos/mqtt/subscription-manager/mqtt_subscription_manager.c

```diff
@@ -149,25 +149,28 @@ static bool matchWildcardsSpecialCases( const char * pTopicFilter,
     bool matchFound = false;
 
     /* Determine if the last character is reached for the topic name, and the
-     * third last character for the topic filter to match for the '#' wildcard. */
+     * third last character is reached for the topic filter. */
     if( ( nameIndex == ( topicNameLength - 1U ) ) &&
         ( filterIndex == ( topicFilterLength - 3U ) ) )
     {
-        /* Determine if the topic filter contains "/#" as the last 2 characters for the '#' wildcard.
-         * The '#' wildcard represents the parent and any number of child levels in the topic name.
-         * For example, the filter "sport/#" matches "sport" as well as "sport/tennis" topics. */
+        /* Determine if the topic filter contains "/#" as the last 2 characters.
+         * The '#' wildcard represents the parent and any number of child levels
+         * in the topic name. For example, the filter "sport/#" matches "sport"
+         * as well as "sport/tennis" topics. */
         matchFound = ( ( pTopicFilter[ filterIndex + 1U ] == '/' ) &&
                        ( pTopicFilter[ filterIndex + 2U ] == '#' ) ) ? true : false;
     }
     else
     {
         /* Determine if the last character is reached for the topic name and,
-         * the second last character for the topic filter to match for the
-         * '+' wildcard. */
+         * the second last character is reached for the topic filter. */
```

Comment:
```suggestion
         * the second to last character for the topic filter. */
```

Reply:
I think you did not want to remove "is reached".

Reply:
That was intentional, I didn't think was necessary to repeat the "is reached" verb for both the 'last character' and 'second to last character' subjects. This is a really minor point though so it's fine if it's changed back.